pkg/provider/aws/plugin/loadbalancer/metadata.go

```diff
@@ -0,0 +1,55 @@
+package aws
```

Comment:
Do I still need this file?

Reply:
I'd leave it for now...  I think there maybe some code that tries to get the region.

Reply:
ok